pkg/allowedcmd/cmd_linux.go

```diff
@@ -0,0 +1,145 @@
+//go:build linux
+// +build linux
+
+package allowedcmd
+
+import (
+	"context"
+	"errors"
+	"os/exec"
+)
+
+func Apt(ctx context.Context, arg ...string) (*exec.Cmd, error) {
+	return validatedCommand(ctx, "/usr/bin/apt", arg...)
+}
+
+func Cryptsetup(ctx context.Context, arg ...string) (*exec.Cmd, error) {
+	for _, p := range []string{"/usr/sbin/cryptsetup", "/sbin/cryptsetup"} {
+		validatedCmd, err := validatedCommand(ctx, p, arg...)
+		if err != nil {
+			continue
+		}
+
+		return validatedCmd, nil
+	}
+
+	return nil, errors.New("cryptsetup not found")
+}
+
+func Dnf(ctx context.Context, arg ...string) (*exec.Cmd, error) {
+	return validatedCommand(ctx, "/usr/bin/dnf", arg...)
+}
+
+func Dpkg(ctx context.Context, arg ...string) (*exec.Cmd, error) {
+	return validatedCommand(ctx, "/usr/bin/dpkg", arg...)
+}
+
+func Echo(ctx context.Context, arg ...string) (*exec.Cmd, error) {
+	return validatedCommand(ctx, "/usr/bin/echo", arg...)
+}
+
+func Falconctl(ctx context.Context, arg ...string) (*exec.Cmd, error) {
+	return validatedCommand(ctx, "/opt/CrowdStrike/falconctl", arg...)
+}
+
+func Falconkernelcheck(ctx context.Context, arg ...string) (*exec.Cmd, error) {
+	return validatedCommand(ctx, "/opt/CrowdStrike/falcon-kernel-check", arg...)
+}
+
+func Gnomeextensions(ctx context.Context, arg ...string) (*exec.Cmd, error) {
```

Comment:
```suggestion
func GnomeExtensions(ctx context.Context, arg ...string) (*exec.Cmd, error) {
```

Not really important, but I'd probably use camel case

Reply:
Ah yeah, I have a bunch of these to update, then. I was waffling on how to capitalize anything with abbreviations in it (e.g. `bputil` -- `Bputil` or `BPUtil` or `BpUtil`) and this was my solution for not having to decide 😅 .

Reply:
I have, at least, now adjusted casing for any command with punctuation in it (so `falcon-kernel-check/Falconkernelcheck` is now `FalconKernelCheck` and `gnome-extensions/Gnomeextensions` is now `GnomeExtensions`.